requestor-tutorials/vm-runtime/computation-payload-manifest.md

```diff
@@ -0,0 +1,278 @@
+---
+description: Computation Payload Manifest description, its schema, and configuration guide
+---
+
+# Computation Payload Manifest
+
+_Computation Payload Manifest_ allows [Requestor](../../introduction/requestor.md) to define application package _Payload_ and allows to set constraints on _Computations_ performed on [Provider](../../introduction/provider.md) node.
+
+Request with a manifest can be [configured in yapapi](#configuration). 
+
+Provider node operator controls what computation can be performed by:
+
+  - [Importing certificates](#3-importing-application-authors-certificates) used to sign App authors' certificates into Provider's [keystore](../../provider-tutorials/provider-cli.md#keystore) (which allows to verify _manifest_ signature)
+
+  - Adding domain patterns to Provider's [domain whitelist](../../provider-tutorials/provider-cli.md#domain-whitelist) (which makes [signature](#2-manifest-signature) optional _manifests_)
+
+## Configuration
+
+Manifest can be configured as a [yapapi.payload.vm.manifest](https://yapapi.readthedocs.io/en/latest/api.html#module-yapapi.payload.manifest) function parameter together with optional manifest signature and [App author's certificate](#certificates).
+
+Check [external API request example](../service-development/service-example-6-external-api-request.md) for details.
+
+## Manifest schema
+
+_Computation Payload Manifest_ must follow a specific [JSON Schema](computation-payload-manifest.schema.json) ([Documentation](computation-payload-manifest.schema.md))
+
+### Schema verification
+
+Manifest can be verified using `jsonschema` library:
+
+```sh
+wget https://handbook.golem.network/requestor-tutorials/vm-runtime/computation-payload-manifest.schema.json
+pip install jsonschema
```

Comment:
Do we even want to keep schema verification guide?
Should the guide be described for both Python (pip) and Nodejs (npm) tools?

Reply:
I'd argue that the verification is independent from the code itself so it can be done with any tool... and of course, a user can use any other tool, so I'd treat the statement above as a suggestion added for user's convenience rather than a requirement...